app/styles/newcomponents/charts.scss

```diff
@@ -0,0 +1,10 @@
+.tooltip {
```

Comment:
This is too global.  It should either be under another top level class `.charts` or it could be made into a mixin which could be imported into `donut-chart.css`.